deployment/load_production_backup.sh

```diff
@@ -41,24 +39,18 @@ then
     exit 1
 fi
 
-[[ -z "$GITHUB_WORKFLOW" ]] && sudo service apache2 stop
```

Comment:
> afaik, we haven't, right?

I mean the actual deployment is not really part of the repo here (and I don't really know what it looks like today myself). I suppose it shouldn't be too different from what we already have, just that instead of activating a venv, a worker has to activate the nix environment or call the wrapped Python interpreter directly.

About keeping it in: See other thread